src/components/MenuItem.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+
+export class MenuItem extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      bread: "Standard",
+      isHot: false,
+      hotPrice: 0.2,
```

Comment:
`hotPrice` could be a constant in a constants file rather than be stored in each items state. It's not something that can be changed and doesn't vary between menu items

Reply:
makes sense - where would that constants file live sir? just in the root? `constants.js` ?

Reply:
exactly that, just in the root is fine 👍